code401challenges/src/main/java/code401challenges/stacksandqueues/Queue.java

```diff
@@ -0,0 +1,54 @@
+package code401challenges.stacksandqueues;
+
+public class Queue<T> {
+    Node<T> front;
+    Node<T> back;
+    T value;
+
+    public Queue(T value, Node<T> front, Node<T> back) {
+        this.value = value;

```

Comment:
Because a `Queue` doesn't have a `value` you can remove all trace of it from the constructor